services/registry/package.json

```diff
@@ -13,36 +13,42 @@
         "test:watch": "jest --watch"
     },
     "dependencies": {
-        "body-parser": "^1.19.0",
-        "express": "^4.17.1",
-        "glob": "^7.1.4",
-        "google-protobuf": "^3.9.0",
-        "graphql": "^14.4.2",
+        "body-parser": "^1.18.3",
+        "express": "^4.16.4",
+        "express-pino-logger": "^4.0.0",
+        "glob": "^7.1.3",
+        "google-protobuf": "^3.7.0",
+        "graphql": "^14.1.1",
         "graphql-tag": "^2.10.1",
-        "grpc": "^1.22.2",

```

Comment:
Why did you downgrade the versions for nearly all modules?

Reply:
Maybe merge issue. I'll fix